packages/@aws-cdk/aws-rds/README.md

```diff
@@ -127,6 +127,38 @@ Creating a "production" Oracle database instance with option and parameter group
 
 [example of setting up a production oracle instance](test/integ.instance.lit.ts)
 
+## Setting Public Accessibility
+
+You can set public accessibility for the database instance or cluster using the `publiclyAccessible` property.
+If you specify `true`, it creates an instance with a publicly resolvable DNS name, which resolves to a public IP address.
+If you specify `false`, it creates an internal instance with a DNS name that resolves to a private IP address.
```

Comment:
Can we explain exactly how you can also control public visibility using `vpcSubnets`? I think it's important here.